student.rb

```diff
@@ -9,4 +9,9 @@ def initialize(classroom, age, name = 'Unknown', parent_permission: true)
   def play_hooks
     '¯(ツ)/¯'
   end
+
+  def classroom=(classroom)
```

Comment:
- [ ] When setting the classroom for a student it has to add it to the classrooms' students. Let's add the getter methods with attr_reader to allow us to access the ` classroom` from outside the scope of the class definition. Kindly add this line 👇🏻 
`attr_reader :classroom` at the top of your Students class